### ISSUE TITLE
UserSettings and UserGameSettings

### ISSUE THREAD

**Issue by hughanderson4** [roblox-ts/services#8]
I need to manipulate some camera settings at certain times.  [Here you can see some discussion](https://devforum.roblox.com/t/how-can-i-force-shift-lock/239206/12?u=firstvertex) of how to achieve that through the `UserSettings.UserGameSettings.RotationType`

I noticed such service is not exposed in roblox-ts/services.  Is there any way it could be added?  Or, did I miss some way that functionality is already exposed elsewhere?

References:
[RotationType](https://developer.roblox.com/en-us/api-reference/property/UserGameSettings/RotationType) - what I want to tweak
[UserGameSettings](https://developer.roblox.com/en-us/api-reference/class/UserGameSettings) - contains the above
[UserSettings](https://developer.roblox.com/en-us/api-reference/class/UserSettings) - contains the above

Thanks!  🍺

**Comment by osyrisrblx** [comment]
This package is purely a convenience thing to use _instead_ of calling `game.GetService()`. For example, you can just fetch RunService via:
```TS
const RunService = game.GetService("RunService");
```

`UserSettings` is actually a weird case. It's a `ServiceProvider` like `DataModel` (game) is, but provides different services. In this case, the service you want from it is `UserGameSettings`. You can get a reference to `UserSettings` by calling the `UserSettings()` global function.

So in summary, to do what you want:
```TS
UserSettings().GetService("UserGameSettings").RotationType
```
